src/main/java/www/purple/mixxy/dao/ComicDao.java

```diff
@@ -101,6 +102,19 @@ public Comic getComic(User user, String slug) {
 
 	    return getComic(user.username, slug);
 	}
+	
+	@SuppressWarnings("null")
+	public List<Comic> getComicsByTags(String[] tags) {
+		if (tags == null)
+			return null;
+
+		List<Comic> comics = new ArrayList<Comic>();
+		for (String tag : tags) {
+			comics.addAll(objectify.get().load().type(Comic.class).filter("tags", tag).list());
```

Comment:
If the comic is filed under more than one of the given tags, it will be duplicated in here.  Can you make this a `Set`?